### DISCUSSION TITLE
[Projects Beta] Provide the ability to hide the "order number" and show the issue number

### DISCUSSION THREAD

**Discussion by paced** [community/community#6691] [Projects and Issues] [answered]
One or both of these will provide the functionality my team is after. We often refer to tickets shorthand by their names e.g., "I'm working on 123."

Thing is, the entire team is used to using the issue number. Not the number next to the board. We would rarely use the issue list (the one GitHub has had for a while) if we now have these Project boards, and of course the numbering can change.

To me, the numbers of ordering are noise: JIRA, with all its flaws, does not provide it. And the issue number even spread across different repos would be quite useful.

**Comment by alexcnichols** [comment, marked as answer]
👋🏼 Improvements for displaying the issue number in the project table, and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/283 :octocat: 

Marking this as a duplicate, in favor of this other discussion:

- https://github.com/github/feedback/discussions/5904

**Reply by paced** [reply]
Cheers!

EDIT: My bad, I didn't know this reply would bump the thread.
